src/components/navBar/demoBasicUsage/script.js

```diff
@@ -8,7 +8,7 @@
     $scope.currentNavItem = 'page1';
 
     $scope.goto = function(page) {
-      console.log("Goto " + page);
```

Comment:
It's bad form to recommend use of globals that Angular provides injectables for. It leads to the spreading of bad habits